src/NzbDrone.Core/HealthCheck/Checks/DownloadClientRootFolderCheck.cs

```diff
@@ -55,9 +57,13 @@ public override HealthCheck Check()
                 {
                     _logger.Debug(ex, "Unable to communicate with {0}", client.Definition.Name);
                 }
+                catch (HttpRequestException ex)
+                {
+                    _logger.Debug(ex, "Unable to communicate with {0}", client.Definition.Name);
+                }
                 catch (Exception ex)
                 {
-                    _logger.Error(ex, "Unknown error occurred in DownloadClientRootFolderCheck HealthCheck");
+                    _logger.Error(ex, "Unknown error occured in DownloadClientRootFolderCheck HealthCheck");
```

Comment:
![image](https://user-images.githubusercontent.com/413544/225771580-e52b67e5-bcaa-49cb-b781-6731cec74aa3.png)

occurred is correct according to google and various links from searching `occured`.